site/app/libraries/FileUtils.php

```diff
@@ -3,6 +3,7 @@
 namespace app\libraries;
 use app\exceptions\FileReadException;
 use app\libraries\Utils;
+use app\libraries\ErrorMessages;
```

Comment:
Remove these two lines. You don't need to add a `use ...` line for things under the same namespace as the file itself.